lib/kubeclient.rb

```diff
@@ -40,7 +40,7 @@ def rest_client
           ssl_client_cert:  @ssl_options[:client_cert],
           ssl_client_key:   @ssl_options[:client_key]
         }
-        endpoint_with_version = @api_endpoint.merge(@api_endpoint.path + '/' \
+        endpoint_with_version = @api_endpoint.merge(@api_endpoint.path \
```

Comment:
> The '\' line continuation is not necessary

actually it is necessary, otherwise I found that rubocop complains, but if I move up the + , I can remove the line continuation and then it works fine.